src/core/Trigger.helper.js

```diff
@@ -13,6 +13,27 @@ export default class TriggerHelper {
     }
   }
 
+  static async defaultCondition () {
```

Comment:
@robertdumitrescu INFO - Not sure how this fits in?  How will it be used for existing implementations?

Reply:
Instead of copying the exact same condition everywhere, we can use this one as a generic one, if there are some complex use cases we can still overwrite it.